packages/components/psammead-navigation/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 6.0.0-alpha.22 | [PR#XXXX](https://github.com/bbc/psammead/pull/XXXX) Use visually hidden text instead of aria label in nav button |
```

Comment:
```suggestion
| 6.0.0-alpha.22 | [PR#2951](https://github.com/bbc/psammead/pull/2951) Use visually hidden text instead of aria label in nav button |
```